server/index.js

```diff
@@ -1,14 +1,93 @@
+/* eslint-disable camelcase */
```

Comment:
Instead of nuking rules for whole files, you can disable them on a line-by-line basis.

https://eslint.org/docs/latest/user-guide/configuring/rules#using-configuration-comments-1